lib/hidapi/udev.py

```diff
@@ -176,6 +176,32 @@ def find_paired_node(receiver_path, index, timeout):
     return None
 
 
+def find_paired_node_wpid(receiver_path, index):
```

Comment:
Can any of this cause an exception?  Perhaps using a try/except would be good defensive practice.

Reply:
I don't see any cause of exception - hidraw device must exist and have usev path. Only if somebody can disconnect receiver exactly during processing.